lib/compress/zstd_lazy.c

```diff
@@ -953,10 +959,11 @@ size_t ZSTD_compressBlock_lazy_extDict_generic(
         U32 current = (U32)(ip-base);
 
         /* check repCode */
-        {   const U32 repIndex = (U32)(current+1 - offset_1);
+        {   const U32 windowLow = ZSTD_getLowestMatchIndex(ms, current+1, windowLog);
```

Comment:
minor : 
I wonder why it's necessary to invoke `ZSTD_getLowestMatchIndex()` at each position, 
as opposed to a constant calculated once ?
I presume this may cost performance (though it's arguably less important for the `_extDict()` variant).

Reply:
We could "invalidate" bad repcodes at the beginning and set them to zero. Then we'd just need to compare against zero. But that is a slightly larger code change.